Development

akrithnayak opened the pull request:
Implemented the original project in object oriented programming. Developers can now send objects from one file to another for further development of this project.

CinnamonHoneyToast committed:
Merge pull request #6 from CinnamonHoneyToast/development

Sync: master to development

akrithnayak committed:
Adding cd.py and implementing oop to all the files

akrithnayak committed:
Deleted unecessary xml files

akrithnayak committed:
Removed a comment in except block in lipy.py

CinnamonHoneyToast commented:
Good work. But need some Modifications:

1. Keep the class name in Upper Camel Case e.g. ClassName.

2. Keep the class names of commands as **full form** of commands e.g. "class ChangeDirectory" for "class cd".

3. Change class lp() to class LiPy.

4. Also, make the **directory_path** as a "self" variable belonging to the LiPy class. This is because, commands like "cd" can change the PATH, but "ls" command will not know the new PATH, nut rather point to the directory which is hard-coded in "dir_path" directory.

5. The command classes such as ChangeDirectory  should inherit LiPy class e.g. ChangeDirectory(Llipy). This is mainly so that the directory_path will be reference by all the commands.

akrithnayak committed:
Made all the necessary changes
1. Changed class names to Upper Camel Case
2. Changed the class names to full form of respective commands
3. Changed the class name from lp() to LiPy()

akrithnayak committed:
Changed the position of object creation in lipy
